README.md

```diff
@@ -179,3 +179,21 @@ Change the `alias` to point to where those packages are actually installed. This
 ```
 
 An alternative to fixing this problem would be to remove aliases altogether and define the dependencies referenced as aliases as dev dependencies instead. [However, that might cause other problems.](https://github.com/palmerhq/tsdx/issues/64)
+
+## Publishing
+To release a new version to npm, we have to signal to CI that a release is needed by changing the version number in the `package.json`.
+There are some basic scripts to help bump the version. This uses `np` to do some basic cleanup tests before allowing the version to be changed.
+Once this passes, commit the package file with the version bump and open a PR. Once merged, this should trigger CI to run an automatic release with that version.
```

Comment:
It seems really easy to forget about this, would it be possible to set up a safeguard to not merge in the same version with different code?

Reply:
CI will require a script to detect a new version has been merged to master. So yea however that script works should not run if no version has changed.  Otherwise its perfectly fine to commit changes to the code without changing the version number. Its only when you want to publish to npm that you should remember to bump the version.